lib/encoder.py

```diff
@@ -234,6 +248,10 @@ def __init__(self, option_set, string, formatter=OptionFormatter()):
       self._HandleFlag(match)
       unparsed = unparsed[match.end():]
       match = re.match(matcher, unparsed)
+    if option_set:
```

Comment:
This is a special case for the option value set copy operator, which needs to create an OptionValueSet that's initialized in a different way than parsing an arugment set. I thought about treating None as a string argument differently from the empty string, but then decided that passing None for the option_set argument was the less gnarly way. See line 315 for the call site.